src/script/microbit-interfacing/Microbits.ts

```diff
@@ -106,6 +110,16 @@ class Microbits {
     }
     return !!this.outputMicrobit;
   }
+
+  public static async dispose(
```

Comment:
I was expecting some per connection clean-up here (e.g. delegating to disponse on the input) but it looks like:
- MicrobitSerial does nothing at construction time
- MicrobitBluetooth registers a gattdisconnect listener but we cache/reuse the MicrobitBluetooth instances for subsequent connections to the same device (as we might still be connecting!) so that's fine too.

Reply:
I was also expecting to clear the gattdisconnect listener, but then realised we don't need to.